bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -201,11 +201,16 @@ object SpProgramFactory {
       timeAccountingRatios(proposal) match {
         case Nil    => None
         case ratios =>
+          val progTime = hoursFromObservations(proposal, _.progTime)
+          val partTime = hoursFromObservations(proposal, _.partTime)
+          require((hrs - (progTime |+| partTime).hours).abs < 1e-1,
+            "Awarded time must be equal to the sum of program time and partner calibration time")
```

Comment:
We have the ratios across partners based on the partner awarded times.  We can sum the program and partner times in the observations and figure out the ratio of program vs. partner.  Then we could apply those two sets of ratios to the sum of the awards to come up with values that by definition match the actual awards?  Is this the concern we're trying to address?  It should work out the same but if for whatever reason the awards don't match the sum of observation times we wouldn't be assigning more or less than the actual awards.